extensions/interactions/TextInput/TextInput.py

```diff
@@ -24,7 +24,7 @@ class TextInput(base.BaseInteraction):
     description = 'Allows learners to enter arbitrary text strings.'
     display_mode = base.DISPLAY_MODE_INLINE
     is_trainable = True
-    _dependency_ids = []
+    _dependency_ids = ['text_input_prediction']
```

Comment:
Seems like @anmolshkl missed this step.

Reply:
@GanitGenius so now prediction service is working fine without any errors, right?

Reply:
yes, i didn't find any errors after making the changes.